### PR TITLE
DOCS-4364 [AOP-2139] Adding info, reorganizing links for install workflow

### PR DESCRIPTION
**Pull request by pablodcar**

### CODE REVIEW

File: modules/ROOT/pages/install-add-tasks.adoc [hunk 1]
```diff
@@ -32,17 +30,18 @@ You must forward logs to a 3rd-party log solution like Splunk. It's not supporte
 
 == Set up SMTP
 
-In order to handle e-mail alerts for both Runtime Manager and API Manager, you must have an SMTP server configured on your network. You must direct your local installation of Anypoint Platform to this server from the Access Management panel.
-
+To handle e-mail alerts for both Runtime Manager and API Manager, you must have an SMTP server configured on your network. You must direct your local installation of Anypoint Platform to this server from the Access Management panel. See xref:access-management-smtp.adoc[Configure SMTP].
```
Comment (fermujica) [inline modules/ROOT/pages/install-add-tasks.adoc]: > See xref:access-management-smtp.adoc[Configure SMTP].

Our link checker is flagging this as a broken link. 
Based on the working link in the `nav.adoc` file,  and the actual name of the file, this line should be:

> See xref:access-management-SMTP.adoc[Configure SMTP].

**OR** We could rename the filename to use all letter case. You call @sleslie23

File: modules/ROOT/pages/config-workflow.adoc [hunk 2]
```diff
@@ -5,12 +5,16 @@ endif::[]
...
+. Ensure that you have configured your system to perform regular backups. See xref:backup-and-disaster-recovery.adoc[About Backup and Recovery]
 
-. Configure an external authentication provider. See xref:install-config-ldap-pce.adoc[To Configure LDAP in Anypoint Platform Private Cloud]
-
-. (Optional) Delete the default Anypoint Platform user created during installation. See xref:install-disable-local-user.adoc[To Delete the Local User after Installing Private Cloud Edition].
+. Perform additional installation tasks as described in xref:install-add-tasks.adoc[To Perform Additional Installation Tasks].
+    * xref:access-management-dns.adoc[Configure DNS]
+    * xref:access-management-security.adoc[Set Certificates]
+    * Log Forwarding
+    * xref:access-management-smtp.adoc[Configure SMTP]
```
Comment (fermujica) [inline modules/ROOT/pages/config-workflow.adoc]: Same issue here: Our link checker is flagging this as a broken link. 
For this to work, you need to either rename the link to 

> See xref:access-management-SMTP.adoc[Configure SMTP].

Or rename the filename to use letter case.

**Review by fermujica — CHANGES_REQUESTED** [review]
I ran a quick quality check and the formatting looks good in all edited documents. 
Overall wording is fine, but one file containing upper cases is failing our broken links test. This needs to be fixed, or our build will fail.

File: modules/ROOT/pages/config-workflow.adoc (hunk 2 above)
Comment (pablodcar) [reply modules/ROOT/pages/config-workflow.adoc]: Done

Comment (fermujica) [reply modules/ROOT/pages/config-workflow.adoc]: This particular link is still being flagged. I see you changed it for the `install-add-tasks.adoc` file, but it needs to be changed here as well.

**Review by fermujica — CHANGES_REQUESTED** [review]
There is one last change for this to go live.

File: modules/ROOT/pages/config-workflow.adoc (hunk 2 above)
Comment (pablodcar) [reply modules/ROOT/pages/config-workflow.adoc]: sorry, I thought I replaced all, Done.

**Review by sleslie23 — APPROVED** [review]

**Review by fermujica — APPROVED** [review]

**Comment by fermujica** [comment]
All tests passed now. Thanks! 
These changes look good to me 👍

**Pull request merged**